assets/patterns/messages/_messages.scss

```diff
@@ -0,0 +1,24 @@
+.messages-table {
+  .messages-table__row {
+    .message-table__data {
```

Comment:
These selectors shouldn't be nested as it makes the specificity too high.

One of the benefits of BEM is that it allows the targeted selection of elements (which reduces specificity) while avoiding naming collisions.